logpolicy/logpolicy.go

```diff
@@ -138,6 +138,149 @@ func runningUnderSystemd() bool {
 	return false
 }
 
+// tryFixLogStateLocation is a temporary fixup for
```

Comment:
say (a bit) what the fixup does and what "dir" and "cmdname" are & are used for?